src/interface/src/app/plan/scenario-details/scenario-details.component.ts

```diff
@@ -79,7 +79,7 @@ export class ScenarioDetailsComponent implements OnInit {
       map((state) => state.currentScenarioId),
       filter((scenarioId) => !!scenarioId),
       switchMap((scenarioId) => {
-        return this.planService.getScenario(scenarioId as string).pipe(take(1));
+        return this.planService.getScenario(scenarioId!).pipe(take(1));
```

Comment:
Where are we getting scenarioId ? . Isn't the id already present on the url?
Can we use `Input()` for this and let the parent component provide it (and mark it as always provided), or  load it from the url?
Also there is a `scenarioId` declared on this component (line 44) , which is not used? Maybe Im getting confused between the two

Reply:
Looking at this component further, I'm not actually sure we still need this page. If it's okay with you, I'll probably keep the changes as is for now just to get this pr through and then look into whether or not we can just delete the component in the next pr

Reply:
sounds good! maybe just comment that on the code somewhere